src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
```

Comment:
Creating a new Random instance every time getRandomColor() is called is inefficient. It is recommended to create a static field for Random and reuse it to generate random numbers.
Instead of returning the color as a string, you can directly return an enum Colors object. This is more type safe and prevents errors due to misspelled color names.